Examples/MAX78000/CNN/kws20_demo-riscv/main_riscv.c

```diff
@@ -475,8 +475,8 @@ void I2SInit()
     /* Initialize I2S RX buffer */
     memset(i2s_rx_buffer, 0, sizeof(i2s_rx_buffer));
     /* Configure I2S interface parameters */
-    req.wordSize = MXC_I2S_DATASIZE_WORD;
-    req.sampleSize = MXC_I2S_SAMPLESIZE_THIRTYTWO;
+    req.wordSize = MXC_I2S_WSIZE_WORD;
+    req.bitsWord = MXC_I2S_BITSWORD_THIRTYTWO;
```

Comment:
@ahmetalincak do we need to initialize `req.SampleSize` here as well?  If we leave it unset I think it will be randomly initialized to any value

Reply:
Good catch. Will resolve in the next commit.